util/plugins/eu.esdihumboldt.util/src/eu/esdihumboldt/util/geometry/WindingOrder.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2016 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     Data Harmonisation Panel <http://www.dhpanel.eu>
+ */
+
+package eu.esdihumboldt.util.geometry;
+
+import com.vividsolutions.jts.algorithm.CGAlgorithms;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.GeometryCollection;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.geom.LinearRing;
+import com.vividsolutions.jts.geom.MultiPolygon;
+import com.vividsolutions.jts.geom.Polygon;
+
+/**
+ * TODO Type description
+ * 
+ * @author Arun
+ */
+public class WindingOrder {
+
+	private static GeometryFactory factory = new GeometryFactory();
+
+	/**
+	 * Identifying order of Geometry object CounterClockwise or not?
+	 * 
+	 * @param geometry Geometry object to identify order.
+	 * @return boolean value, true, if order is counterclockwise, else false if
+	 *         order is clockwise.
+	 */
+	public static boolean isCounterClockwise(Geometry geometry) {
+		if (geometry == null)
+			return false;
+
+		// Get coordinates of geometry
+		Coordinate[] coordinates = geometry.getCoordinates();
+		// Using algorithm get order of geometry
+		boolean orientation = false;
+		try {
+			orientation = CGAlgorithms.isCCW(coordinates);
+		} catch (IllegalArgumentException ex) {
+			System.err.println("Illegal Argument");
```

Comment:
It makes sense to propagate the exception instead of silently ignoring it. Otherwise you will return information that was not checked in any way. Thus a caller would never be sure if the result from the `isCounterClockwise` is correct or if there was an internal error.